### PR TITLE
YUPANA-86 - add log statements & try-except in gc processor

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: yupana/processor/garbage_collection.py [hunk 1]
```diff
@@ -62,31 +67,39 @@ async def run(self):
...
-                (report_total, report_slice_total, int(ARCHIVE_RECORD_RETENTION_PERIOD))))
-        else:
-            LOG.info(
-                format_message(
+        try:
+            current_time = datetime.now(pytz.utc)
+            created_time_limit = current_time - timedelta(seconds=ARCHIVE_RECORD_RETENTION_PERIOD)
+            # we only have to delete the archived reports because deleting
+            # an archived report deletes
+            # all of the associated archived report slices
```
Comment (gnurag) [inline yupana/processor/garbage_collection.py:77]: Remove the newline in this commented part, so its back to how it originally was?

File: yupana/processor/garbage_collection.py [hunk 2]
```diff
@@ -62,31 +67,39 @@ async def run(self):
                     % int(GARBAGE_COLLECTION_INTERVAL)))
             await asyncio.sleep(GARBAGE_COLLECTION_INTERVAL)
 
-    @DB_ERRORS.count_exceptions()
```
Comment (gnurag) [inline yupana/processor/garbage_collection.py:65]: Curious - is this decorator no longer required?

**Review by gnurag — COMMENTED** [review]
Additional error handling looks reasonable.

File: yupana/processor/garbage_collection.py (hunk 2 above)
Comment (kgaikwad) [reply yupana/processor/garbage_collection.py:65]: Yes, no longer required as I have added [logic](https://github.com/quipucords/yupana/pull/369/files#diff-a7b1037f11cf2cb717af98d848a00ac1da95f85e2d3d526527b59025010eb721R97) to increment DB_ERRORS under except block where I have mentioned `Exception` class. 
Just for reference:
- https://prometheus.io/docs/instrumenting/writing_clientlibs/#counter
- https://github.com/prometheus/client_python#counter

File: yupana/processor/garbage_collection.py (hunk 1 above)
Comment (kgaikwad) [reply yupana/processor/garbage_collection.py]: Modified removed new line and adjusted text string as I shifted code under try block it was giving flake8 warning for first line.

Comment (gnurag) [reply yupana/processor/garbage_collection.py]: Thanks 👍

**Review by gnurag — APPROVED** [review]
LGTM

**Comment by kgaikwad** [comment]
Thank you @gnurag for review! Merging this change.

**Pull request merged**